### PR TITLE
Front homepage

### PR DESCRIPTION
**Pull request by bisroka**
I have finished responsive view for tablets and desktop

### CODE REVIEW

File: front-end/src/Components/ProductCard.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+const ProductCardWrapperStyled = styled.section`
+  display: block;
+  position: relative;
+  border-style: solid;
+  border-width: 0 100vw 40vh 0;
+  border-color: transparent #97bf04 transparent transparent;
+`;
+
+const ProductCardStyled = styled.div`
```
Comment (Imbruced) [inline front-end/src/Components/ProductCard.js:14]: Can we move all styling stuff to css files ?

File: front-end/src/Components/SearchBar.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+          placeholder="Wpisz nazwę produktu"
+        />
+      </form>
+    </SearchBarStyled>
+  );
+};
+
+export default SearchBar;
+
+// class SearchBar extends Component {
```
Comment (Imbruced) [inline front-end/src/Components/SearchBar.js:52]: If we do not use that, remove.

File: src/main/kotlin/pl/diet/company/dietapp/controller/ProductController.kt [hunk 3]
```diff
@@ -9,8 +9,11 @@ import pl.diet.company.dietapp.service.ProductQuery
 
 import pl.diet.company.dietapp.service.ProductReader
 
+import org.springframework.web.bind.annotation.*
```
Comment (Imbruced) [inline src/main/kotlin/pl/diet/company/dietapp/controller/ProductController.kt:12]: Please rebase this branch with current master version.

File: front-end/src/Components/Menu.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+import { device } from "../utils/device";
+
+const MenuStyled = styled.div`
```
Comment (Imbruced) [inline front-end/src/Components/Menu.js:6]: Can we create sub modules based on components functionality ? I am afraid that we end up with  all of the components in the same place.

File: front-end/src/assets/svg/output.svg [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Imbruced) [inline front-end/src/assets/svg/output.svg:1]: Do we need this default files ?

File: package-lock.json [hunk 6]
```diff
@@ -0,0 +1,313 @@
+{
+  "requires": true,
```
Comment (Imbruced) [inline package-lock.json:2]: Can you remove locked package file ? I think this is not required.

Comment (Imbruced) [reply package-lock.json:2]: Also we can start clearing project. Do we need **node_modules** at root directory ?

**Review by Imbruced — CHANGES_REQUESTED** [review]

File: front-end/src/Components/ProductCard.js (hunk 1 above)
Comment (bisroka) [reply front-end/src/Components/ProductCard.js:14]: i used styled-components library so far in this project, but we can decide for css modules if you want and I will refactor all files

File: front-end/src/Components/SearchBar.js (hunk 2 above)
Comment (bisroka) [reply front-end/src/Components/SearchBar.js:52]: ok :)

File: src/main/kotlin/pl/diet/company/dietapp/controller/ProductController.kt (hunk 3 above)
Comment (bisroka) [reply src/main/kotlin/pl/diet/company/dietapp/controller/ProductController.kt:12]: ok

File: front-end/src/assets/svg/output.svg (hunk 5 above)
Comment (bisroka) [reply front-end/src/assets/svg/output.svg:1]: it is our exit icon :)

**Comment by Imbruced** [comment]
WIP

**Comment by bisroka** [comment]
Refactor done, review that, pls

**Review by bisroka — COMMENTED** [review]
now its ok

**Pull request merged**